components/ca-radio-group.js

```diff
@@ -66,26 +70,26 @@ define([], () => {
 
         /**
          * Sets data
-         * @param {array} data data
+         * @param {object} data data
          * @returns {void} void
          */
         set data(data) {
```

Comment:
yup, I think thats a discussion for when we can start refactoring - what does a common component do, what are the standard methods and attributes etc.

I think using 'schema' as an attribute for this is misleading - the ca-form takes a schema, the controls actually take an array or object of data.  src is incorrect too, it's not a uri, which src usually is.  I think 'data' is most appropriate here, but as with all things I can be easily pursuaded by a better option  :)